src/websocket-client.js

```diff
@@ -64,8 +64,8 @@ class WebSocketClient extends Emitter {
 
       if (!this.initialized) {
         this.handleMessage(message, this.socket);
-      } else {
-        this.eventEmitter.emit(message.name, message);
+      } else if (message.op === 'event') {
+        this.eventEmitter.emit(message.name || message.data.name, message.data || message);
```

Comment:
Should we support V1 and V2 in parallel ? if yes:

Passing `&version=2` is telling the server the maximum version supported. It's up to the server to choice the version that both end must use.

When you receive the first `init` message, you have the version that both end must use.

You can take a look here:
https://github.com/TinxHQ/nestbox-ui-ng/pull/213/files#diff-15613bc5845576e1c2bfdef7efdcaf7eR42-R68

Reply:
Yes but I prefer let the user choose the version he want to use.